pom.xml

```diff
@@ -9,15 +9,14 @@
     <relativePath />
   </parent>
   <artifactId>tuleap-git-branch-source</artifactId>
-  <version>1.0.8-SNAPSHOT</version>
+  <version>1.0.9-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.138.4</jenkins.version>
+    <jenkins.version>2.204.2</jenkins.version>
```

Comment:
> Updating targeted Jenkins version as a vulnerability disclosure has been released

While I'm all in for pushing users to update their stuff (especially Jenkins instances) we do not have enough impact as plugin creators to force administrators to update to a version released 2 days ago. It's not our role to do that and it will prevent Enterprise ™ users to use the plugin.

I suggest we bump to 2.150 (maybe 2.150.3) which is the oldest LTS version currently supported by the update center (see the content of https://updates.jenkins.io/).